tests/display.rs

```diff
@@ -1278,6 +1284,154 @@ mod enums {
                 }
             }
         }
+
+        mod shared_format {
+            use super::*;
+
+            mod single {
+                use super::*;
+
+                #[derive(Display)]
+                #[display("Variant: {_variant}")]
+                enum Enum {
+                    #[display("A {_0}")]
+                    A(i32),
+                    #[display("B {}", field)]
+                    B {
+                        field: i32,
+                    },
+                    C,
+                }
+
+                #[test]
+                fn assert() {
+                    assert_eq!(Enum::A(1).to_string(), "Variant: A 1");
+                    assert_eq!(Enum::B { field: 2 }.to_string(), "Variant: B 2");
+                    assert_eq!(Enum::C.to_string(), "Variant: C");
+                }
+            }
+
+            mod transparent {
+                use super::*;
+
+                #[derive(Display)]
+                #[display("{_variant}")]
+                enum Enum {
+                    #[display("A {_0}")]
+                    A(i32),
+                    #[display("B {}", field)]
+                    B {
+                        field: i32,
+                    },
```

Comment:
I would want to see a transparent field as well, that correctly passes on the format options. It should 

```
// adding this variant the list
#[display("{:b}")]
Binary(i32)

// and then doing this test
assert_eq!(format("{:08}", Enum::Binary(4)), "00000100");
```

Reply:
Done.